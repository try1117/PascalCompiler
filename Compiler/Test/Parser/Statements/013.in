program test;
var a: integer;
begin
  if 1.0 then a := 1;
end.