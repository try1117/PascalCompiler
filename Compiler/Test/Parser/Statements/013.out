Lexical exception in position (4,10) - Incompatible types, expected "Integer" but found "Double"
